thirdparty/README.md

```diff
@@ -297,6 +298,7 @@ File extracted from upstream release tarball `mbedtls-2.16.0-apache.tgz`:
 - Applied the patch in `thirdparty/mbedtls/1453.diff` (PR 1453). Soon to be merged upstream. Check it out at next update.
 - Applied the patch in `thirdparty/mbedtls/padlock.diff`. This disables VIA padlock support which defines a symbol `unsupported` which clashses with a symbol in libwebsockets.
```

Comment:
clashses -> clashes.

Reply:
That's not from this commit, but sure, I can fix that too.

Reply:
Fixed.